Syllabus/_child_assignments.Rmd

```diff
@@ -0,0 +1,32 @@
+<!-- This document contains assignment details that appear multiple places. -->
+
+## Grading
+***
+
+1. Attendance/Participation: 20% (Please let the instructors know if attendance will be difficult for you.)
+2. Homework: 3 x 15%
+3. Final "Project": 35%
+
+All assignments are due Wednesday, Jan 26, 2022 at 11:59pm EST.
+
+**Note**: only people taking the course for credit must turn in the assignments. However, we will evaluate all submitted assignments in case others would like feedback on their work.
+
+
+## Submitting Assignments
+***
+
+Submit each assignment to the designated [Drop Box on CoursePlus](https://courseplus.jhu.edu/core/index.cfm/go/db:main/coID/16733/).
```

Comment:
```suggestion
Submit each assignment to the designated [DropBox on CoursePlus](https://courseplus.jhu.edu/core/index.cfm/go/db:main/coID/16733/).
```

Reply:
I checked this on CoursePlus and it's not actually DropBox but a thing on CoursePlus called Drop Box.